src/map/lua/lua_item.cpp

```diff
@@ -149,6 +149,16 @@ bool CLuaItem::isSubType(uint8 subtype)
     return m_PLuaItem->isSubType(static_cast<ITEM_SUBTYPE>(subtype));
 }
 
+bool CLuaItem::isRare()
+{
+    return m_PLuaItem->getFlag() & ITEM_FLAG_RARE;
```

Comment:
suggestion: collapse these to one function and pass the enum or values in as input so one can check either flag or even multiple as mask bits.

See canUseMisc for an example of what I mean.

Reply:
+1 for this, will change when I have time